craft_providers/lxd/lxc.py

```diff
@@ -792,7 +794,7 @@ def list(
 
     def list_names(
         self, *, project: str = "default", remote: str = "local"
-    ) -> List[str]:
+    ) -> builtins.list[str]:
```

Comment:
question: is this a different type on purpose here?

Reply:
I'm not sure why ruff decided to do this rather than just using `list`, but they are the same.

<img width="227" height="109" alt="image" src="https://github.com/user-attachments/assets/ba66a69b-43ae-4270-8e00-83418a38c85a" />

Reply:
Ahh, update: it did that because we have a `list` method above, so we need to disambiguate in the namespace.